tox.ini

```diff
@@ -4,6 +4,7 @@ envlist = py37, py38, py39, py310, py311, pypy37, pypy38, pypy39, checks, docs
 
 [testenv]
 description = Run the pytest suite
+deps = freezegun>=1.2.1
```

Comment:
this would probably be better configured in pyproject.toml amongst the testing extras ;)

Reply:
Thanks, first time using poetry. Moved.